src/main/scala/controllers/ApplicationPreviewController.scala

```diff
@@ -63,6 +63,26 @@ class ApplicationPreviewController @Inject()(actionHandler: ActionHandler, appli
 
   def gatherApplicationDetails(id: ApplicationId): Future[Option[ApplicationDetail]] = applications.detail(id)
 
+    def viewQuestions(id: ApplicationId, sectionNumber: Int) = Action.async { request =>
```

Comment:
This shouldn't be in ApplicationPreviewController. The questions come from application_form_section which exists independently of any particular application.